opera-project/admin-bundle/0.1/config/packages/opera_easy_admin.yaml

```diff
@@ -0,0 +1,15 @@
+## This is a sample easy admin configuration
+## Copy Page Section to your own easy admin configuration
+#
+imports:
+    - { resource: "@OperaAdminBundle/Resources/config/easy_admin.yaml" }
+
+# easy_admin:
```

Comment:
These comments needs to be removed. They do not bring any value. 

Consider adding a link to your documentation.